cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/KafkaCruiseControlUtils.java

```diff
@@ -541,4 +547,20 @@ public static void sanityCheckNonExistingGoal(List<String> goals, Map<String, Go
 
     return balancednessCostByGoal;
   }
+
+  /**
+   * Reads the configuration file, parses and validates the configs.
+   * @param propertiesFile is the file containing the Cruise Control configuration.
+   * @return a parsed {@link KafkaCruiseControlConfig}
+   * @throws IOException if the configuration file can't be read.
+   */
+  public static KafkaCruiseControlConfig readConfig(String propertiesFile) throws IOException {
+    Properties props = new Properties();
+    try (InputStream propStream = new FileInputStream(propertiesFile)) {
+      props.put("config.providers", "env");
+      props.put("config.providers.env.class", EnvConfigProvider.class.getName());
```

Comment:
These hardcoded config names and the way the user is expected to name their environment variables to pass them to CC is not very clear. Would it help if we (1) refer to the following names rather than hardcoding these configs:
1. `AbstractConfig#CONFIG_PROVIDERS_CONFIG`, and
2. `AbstractConfig#CONFIG_PROVIDERS_PARAM`

and (2) provide a brief section in `Configurations.md` regarding how to name env variables to pass them to CC?

Reply:
`AbstractConfig#CONFIG_PROVIDERS_PARAM` seems to be private but I'll resolve the other points.